Frontend/app/tests/example.ts

```diff
@@ -0,0 +1,31 @@
+// A sample QUnit test
+
+/*QUnit.test("equal test", function (assert) {
+  assert.equal( 0, 0, "Zero, Zero; equal succeeds" );
+  assert.equal( "", 0, "Empty, Zero; equal succeeds" );
+  assert.equal( "", "", "Empty, Empty; equal succeeds" );
+  assert.equal( 0, false, "Zero, false; equal succeeds" );
+});*/
+
+var participants = require("../pages/enterprises/participants.js");						//Access Participant variables and functions
```

Comment:
Given that we don't have a pages folder, let alone a participants file, how does this work?